File: Programming Basics with JS/T03 - Conditional Statements/exercise/05. Godzilla vs Kong.js
function movie(input) {
    let budget = Number(input[0]);
    let stat = Number(input[1]);
    let priceForDressOfOneStat = Number(input[2]);
    let priceOfDecor = budget * 0.10;
    let totalPriceForStat = stat * priceForDressOfOneStat;
    let totalPriceOfTheMovie = totalPriceForStat + priceOfDecor;

    if (stat > 150) {
        totalPriceForStat = totalPriceForStat * 0.9;
    }

    totalPriceOfTheMovie = totalPriceForStat + priceOfDecor;
    let difference = Math.abs(totalPriceOfTheMovie - budget);

    if (totalPriceOfTheMovie <= budget) {
        console.log("Action!");
        console.log(`Wingard starts filming with ${difference.toFixed(2)} leva left.`);
    } else {
        console.log("Not enough money!");
        console.log(`Wingard needs ${difference.toFixed(2)} leva more.`);
    }
}
movie(["5", "5", "5"]);

File: Programming Basics with Python/T03 - Conditional Statements/exercise/05. Godzilla vs. Kong.py
budget = float(input())
walkers = int(input())
costumes_price = float(input())

stage = budget * 0.10
costumes_cost = walkers * costumes_price
discount = costumes_cost * 0.10

total_cost = stage + costumes_cost

if walkers >= 150:
    total_cost = total_cost - discount

diff = abs(budget - total_cost)

if total_cost <= budget:
    print('Action!')
    print(f'Wingard starts filming with {diff:.2f} leva left.')
else:
    print('Not enough money!')
    print(f'Wingard needs {diff:.2f} leva more.')